pom.xml

```
@@ -64,7 +64,7 @@
     <properties>
         <revision>2.40</revision>
         <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.138.4</jenkins.version>
+        <jenkins.version>2.150.1</jenkins.version>
```

I'd go to the latest release of that LTS line, so 2.150.3:

```suggestion
        <jenkins.version>2.150.3</jenkins.version>
```